SimPPS/RPDigiProducer/src/RPHitChargeConverter.cc

```diff
@@ -0,0 +1,33 @@
+#include "SimPPS/RPDigiProducer/interface/RPHitChargeConverter.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "SimPPS/RPDigiProducer/interface/RPLinearChargeDivider.h"
+#include "SimPPS/RPDigiProducer/interface/RPLinearChargeCollectionDrifter.h"
+#include "SimPPS/RPDigiProducer/interface/RPLinearInduceChargeOnStrips.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+
+RPHitChargeConverter::RPHitChargeConverter(const edm::ParameterSet &params, CLHEP::HepRandomEngine& eng, RPDetId det_id)
+  : params_(params), det_id_(det_id)
+{
+  verbosity_ = params.getParameter<int>("RPVerbosity");
+  theRPChargeDivider = new RPLinearChargeDivider(params, eng, det_id);
+  theRPChargeCollectionDrifter = new RPLinearChargeCollectionDrifter(params, det_id);
+  theRPInduceChargeOnStrips = new RPLinearInduceChargeOnStrips(params, det_id);
```

Comment:
Could these be regular members instead of pointers?

Reply:
std::unique_ptr<> may be used

Reply:
Local members should have "_"